porter/v1/system_service.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package porter.v1;
+
+import "porter/v1/prometheus_alerts.proto";
+
+// SystemService identifies a system service by its name, namespace and the type of k8s object it runs
+message SystemService {
+  // name is the name of the service
+  string name = 1;
+  // namespace is the namespace where it is deployed
+  string namespace = 2;
+  // involved_object_type is the k8s object type the service runs
+  InvolvedObjectType involved_object_type = 3;
```

Comment:
we should try to not expose the k8s details if we don't have to (why would the dashboard have to share what namespace a system component is in, or what type of object it is?)

Reply:
This is for internal use for now. We want to have as much information that helps in debugging as much as possible.